lib/commands/diff.js

```diff
@@ -7,6 +7,7 @@ const pacote = require('pacote')
 const pickManifest = require('npm-pick-manifest')
 const log = require('../utils/log-shim')
 const readPackageName = require('../utils/read-package-name.js')
```

Comment:
It seems like this is used a few other places in this file. Can it be removed from those too?

Reply:
I thought i got them all. It's very concerning that tests still pass here.

Reply:
also i think the actual util file didn't get deleted, only the test

Reply:
This is why we review.  Removed.

Reply:
at least that explains why the tests passed previously 😅